client/package.json

```
@@ -67,6 +67,7 @@
     "scroll-tabs": "^1.0.1",
     "semver-compare": "^1.0.0",
     "sourcemapped-stacktrace": "^1.1.11",
+    "tiny-svg": "^3.1.3",
```

```suggestion
    "tiny-svg": "^3.1.3",
```

Should we not just `npm update tiny-svg` to update the dependency (nested)?

---

Feels odd to introduce this here (unless we use it within the modeler).

---

(We do use it, but it is pulled in through `bpmn-js`, so I'd expect to simply bump the nested dependency).

---

I pulled it because we directly depend on it. I should've commented on that.